lib/Log/OwncloudLog.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * @author Piotr Mrowczynski <piotr@owncloud.com>
+ *
+ * @copyright Copyright (c) 2017, ownCloud GmbH
+ * @license AGPL-3.0
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\Diagnostics\Log;
+
+/**
+ * logging utilities
+ *
+ * Log is saved at data/diagnostics.log (on default)
+ */
+
+class OwncloudLog {
+	static protected $logFile;
+
+	/**
+	 * Init class data
+	 */
+	public static function init() {
```

Comment:
Hmm, to be honest tried to follow the "original" OwnCloud logger, but maybe ye, as utility class it might not make that much sense https://github.com/owncloud/core/blob/2b21ad761abff580cb420c6e5fb1df83c81e9f1a/lib/private/Log/Owncloud.php#L39-L39